modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -111,12 +111,18 @@ definition {
 	}
 
 	@description = "Verify that the user can add Nodes into diagram by drag&drop: Join"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanAddJoinNode {
+		property test.name.skip.portal.instance = "ProcessBuilderKaleoDesignerReact#CanAddJoinNode";
+
+		Workflow.openWorkflowListView();
 
-		// TODO CanAddJoinNode pending implementation
+		Workflow.addNewDefinition(workflowDefinitionTitle = "New Workflow Title");
 
+		ProcessBuilderKaleoDesignerReact.dragSidebarNodeAndMove(
+			nodeLabel = "Join",
+			nodePosition = "-20,-40",
+			nodeType = "join");
 	}
```

Comment:
It would be interesting to add a `viewPanelNodePresent` to check the node in the diagram.

Reply:
Is there still a possibility to change? Yes, I missed that detail.
anything I can go back and put this step to the test to be more complete, ok